test/Sentry.NLog.Tests/SentryTargetTests.cs

```diff
@@ -488,17 +488,27 @@ public async Task LogManager_WhenFlushCalled_CallsSentryFlushAsync()
         public void InitializeTarget_InitializesSdk()
         {
             _fixture.Options.Dsn = null;
-            _fixture.Options.Debug = true;
             _fixture.SdkDisposeHandle = null;
             _fixture.Options.InitializeSdk = true;
-            var logger = Substitute.For<IDiagnosticLogger>();
 
-            logger.IsEnabled(SentryLevel.Warning).Returns(true);
-            _fixture.Options.DiagnosticLogger = logger;
+            var logwriter = new System.IO.StringWriter();
 
-            _ = _fixture.GetLoggerFactory();
-            logger.Received(1).Log(SentryLevel.Warning,
-                    "Init was called but no DSN was provided nor located. Sentry SDK will be disabled.", null);
+            try
+            {
+                InternalLogger.LogWriter = logwriter;
+                InternalLogger.LogLevel = LogLevel.Debug;
+
+                _ = _fixture.GetLoggerFactory();
+
+                var logoutput = logwriter.ToString();
+                Assert.Contains("Init was called but no DSN was provided nor located. Sentry SDK will be disabled.", logoutput);
+            }
+            finally
+            {
+                _fixture.Options.Debug = false;
```

Comment:
There's a new instance of `_fixture` per test so there's no need to reset values.

Reply:
Ok will remove.